@navikt/core/tokens/types.ts

```diff
@@ -0,0 +1,219 @@
+export type ColorTheme = "light" | "dark";
+
+export type ColorRoles =
+  | "neutral"
+  | "accent"
+  | "success"
+  | "warning"
+  | "danger"
+  | "info"
+  | "brand-magenta"
+  | "brand-beige"
+  | "brand-blue"
+  | "meta-purple"
+  | "meta-lime";
+
+export type GlobalColorScale =
+  | "100"
+  | "200"
+  | "300"
+  | "400"
+  | "500"
+  | "600"
+  | "700"
+  | "800"
+  | "900"
+  | "1000"
+  | "000"
+  | "100A"
+  | "200A"
+  | "300A"
+  | "400A";
+
+export type GlobalColorKeys =
+  | `${Extract<ColorRoles, "neutral">}-${Extract<GlobalColorScale, "000">}`
+  | `${ColorRoles}-${Exclude<GlobalColorScale, "000">}`;
+
+/* Semantic tokens */
+
+export type DefaultBgKeys =
+  | "default"
+  | "input"
+  | "raised"
+  | "sunken"
+  | "overlay";
+
+export type StaticBgKeys =
+  | ColorRoles
+  | `${ColorRoles}-moderate`
+  | `${ColorRoles}-moderateA`
+  | `${ColorRoles}-strong`
+  | `${ColorRoles}-raised`;
+
+export type StatefulBgKeys =
+  | `${ColorRoles}-hover`
+  | `${ColorRoles}-hoverA`
+  | `${ColorRoles}-moderate-hover`
+  | `${ColorRoles}-moderate-hoverA`
+  | `${ColorRoles}-moderate-pressed`
+  | `${ColorRoles}-moderate-pressedA`
+  | `${ColorRoles}-strong-hover`
+  | `${ColorRoles}-strong-pressed`
+  | `${ColorRoles}-raised-hover`;
```

Comment:
`StatefulBgKeys` doesn't seem to be in use anywhere? 🧐

Reply:
Thats right, there us currently no need for the type externally. Internally it should really be used in `semantic-roles.ts`, but haven't found a nice way to incorporate the types here without adding lots of complexity, so still using `satisfies StyleDictionaryTokenConfig<"color">` there.